automation/android/espresso/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
```

Comment:
I believe that, when I tried to do upgrade this, either the command-line build or the android studio build broke.  Would you mind giving both a try before pushing this?

Reply:
![image](https://user-images.githubusercontent.com/1173057/51934031-ab8fab80-23d0-11e9-8f78-5c6d51d5dc81.png)
![image](https://user-images.githubusercontent.com/1173057/51934061-bfd3a880-23d0-11e9-95fe-1dd6889aeafc.png)
They are passing for me. Ideally we should be able to rely on CI to catch broken builds. If there's not a CI job to detect regressions, let's set one up.

Reply:
👍